project/plugins/aws/__init__.py

```diff
@@ -80,29 +93,42 @@ def removeUser(email, configMap,allPermissions, plugin_tag):
         response = client.list_groups_for_user(UserName=username)
         groups = response.get('Groups')
         for group in groups:
-            client.remove_user_from_group(GroupName=group.get('GroupName'), UserName=username)
+
+             try:
+
+                 response = client.remove_user_from_group(GroupName=group.get('GroupName'), UserName=username)
+
+             except ClientError:
+                 log = (plugin_tag + 'Could not remove user ' + username + ' from the group')
+                 instruction = log
 
         # remove access keys
         response = client.list_access_keys(UserName=username)
         keys = response.get('AccessKeyMetadata')
-        for key in keys:
-             client.delete_access_key(
-                UserName=username,
-                AccessKeyId=key.get('AccessKeyId')
-            )
 
-        # delete login profile
         try:
 
-            log = plugin_tag+': User ' +username+ '. ' + 'profile has been deleted\n'
+            for key in keys:
+                 client.delete_access_key(
```

Comment:
Does this return anything? Should we assign it to a variable and check the value?